assets/stylesheets/garnett.scss

```diff
@@ -7,38 +7,50 @@
aK7DiZ+SUfcgW1pP3NkWU8Cw4zzmXXOfc3ygsHVBQTUWB8Iu8qyNqqsqoq+pSZbTAX2ggB5FVtUAL5lwLy7VKrDWVBYX1b98Gs2qMCDWad12BgzlNpU40mGbQRhpOR2kyzaGW1B+8WxnZWxFkVy72s519jGJ+r5CGvYCuCcDcL9J5GYt0mY8UccMqw5D6CcHa/5cAAAB4AV3QA5SdVxAA4G/exmXsl383tm3btlU3dWOWYW3btm03NuvG5h62c8eeixQyUEiGyJY5MpaghUQOaTU1NsNDnvK816NhjI4ZsSyOp0qmGqY+S61Nz0/vTvImhZISSTrJSioktZImSbvk6cysrALHjyNRSxP3e8SzXvZmNI5xJ+sPpYqdrP80tSI9K73rZH3BpFhSKkmy6xv/p36d/8HxKXDs22Ptj82FjaM35suWmRvTGwuflPk2PL6h/frmQgsMsckOexwIJ6d2ITUttQBILUgtyZaLUsuAbBsQk6Jm1Ira0SvqRN2oF2WiXNSPDtEgSkXDaBSNo0l0jE7ROZpGM+F0ZyiosDIS5VVUW131NdZES5101kUP/Qw0xliTTHZBJNE8ykf/KBkDzHajhW5yi/uyf+sxz3vRy173hnd95nNf+MZPfrXRJtvt8FdkRpeoEF1jcrSI3tEn+sboGBMDY6ydUTHGxaAYH4PNignRMobYFVWjWBSPEtFKSoYccsoltynOVERRxRRXSWVVVHWpplpprY222kU3lxlksCGGOscVdptjrnnmW+Aaeyxyq9vc7g6Pe8KTnrLPm97zvg986KPobr/lVlhpld8c9Ls8rpXXdfK53lkWO9sS+S1VwDKF3OxKh5Rwp5LuUsrdSrtH2r0yPSDLg8p6SDkPq+BRVzmsmqdV94wanlXTc2o54mpH1fGCel7SwCsaelUjr5nqmGmOa+Ytzb2thXeC6RHa+1gHn+joUzMipasvdfOV7r7W07d6+U5v3+vjB/39bGZkGGa14dYYYa2R1hllvdE2GGezc/3hPBf620X+cbF/neYGl9jrcgf09aMBfjHeFhNsNdE25/szKkW1yIrKUT3KRpWocQKDFcnTeAEUwQOwEFAAALBnK9u2bdu2bds2D9m2Dtm2bX9mcwP/ZQR5QUlQFTQEbUFPMBRMBEfAFWhhaVgNNoLtYC84DK6FO+Ft+BzGwp9IoPgoNcqOCqPyqBNagjagPegIOo++YYqT48w4Py6Nq+PGuD3ujYfhSXg93o2f4EhCSRySnGQmDchQMpHMJUvIBnKfvCYfKaCGJqbpaW5anPam6+l9+pp+oH9YBVaHtWCdWX82ii1i69hOdo+9Yh/YH655Ip6O5+LFeCU+nW/k1/gjHsG/CCzyikaineglholJYo3YIQ6IMxJJJ8vIWXKRvCxvq3gqlaqsuqsV6pb6obPonnqIHq9n6YV6jd6u9+mTJq3JYzqZ+WaZ2Wj2mMPmnOU2nq1o69jmtpPta0fYyXaefWqjXDI3xc13y9xGt8e9dO99Yp/e5/JFfQVf2zfzHX0fv8Jv9jf9E/81kOBD0tA+zA/LwsawJxwO0eFvS/AAM0QUAAA4Ww+npzPy1ZCtG/KUh2zbtm3btqaM3zYyhmx832dQAlQGBLggBI1AWzAHXAQ54DX4BIvD1rAj7A77wZFwCtwJ78MEVBJVQRR5qA5qjNqhLqgXGoiOowfoIy6OK2GCPdwFz8JL8Qa8Gx/D93A8zsIvpVDqIR2XgczlAfIp+ar8VWmijFEWK+uVa0qG8ly11XHqLjVR/ahF2gztDKlMBpLJZCnZRO6SDFqORrQ/XUdT6HvWlM1ha9gedobdYPdZEsvnZTnlA/koPplv4ydEEUFFLdFB9BGTxTJxVFwVyeKVTvXpepKeb5Q2iBEazYx+xkLjkdnQjMx+5nDznBlrfrWAZVrHrdtWvPXM+mqXsWvbkT3Onmuvst86dZ3Bzlan0PnslnZrug3dyO3vznDXuVfcNK+0x7zQ2+4X83v48/39fuH/X4JKQf2gazAoOPALEkrNEAAAeAGcmQVgW0fS+HffE9gCW9KzJMuSJUuywCTZliWZWWaOY8mcxHG43CRlpiTFpMyQfr1y6yZ3/zIc9LvS8fXK7fGlzHR677+772klJ+5HBcPPTzOzM7O7M/MAA7IBYO5jfgNyAQeAx+dUmo3ucJQLcWajM8yyIRZ+38ALjSMvtP5r5FAr//3mo97aVrW0pC9y6phbk2t0ziKmlX8UdgNBEGWxv2C8AAIAlHAC9IH7V+CT4KwMvkS5GoyDA5RfTfk8iIM7KT+H8jhYA/ZR/lvKZ8F6cA/li5QnwGpwDeXXUj6N9D6AfSGEiS+KQDEAcpc3XBOJLvOIW6Ew5plC1ZGoUaEgzvlHRW1Tb68AUg7aWVtuLdAaNdrSp5GjXvR7HE3+tzK85Q+ZjUZWVpylYsuw21LWyB6UrPGJ1hMvnQuvBiywoL+vQVZxwA58KEZ5CrczHNX7ws5qkzFPoTSaTB5kUrgmwLjdevKT1+1m1px/Mf/CZBwWXHDWwPimWf60ePuU0d/gmThqND4xdNxTM+ucemabwTk+0zitVWqnTmsarojaVck9LSPjQwDZYhe+YPqZt4AT6UQrjoaUCoXbFWDCNc1MqNpkNrt9Xq/bhT0CTyqPmQqDMt9i78CZiaqGtWd0ti04ZcEifcxzwvatRbnqKw7qDKWJi9du3pvw5aoeujBbe+ctt98rCEiLHmtBq9ei1Svg1CwgFOkmEdVKGXMVeJTypyhXg03gGcovp3wGbAdXUf4s5RqwDRxEkTYjn96OfKrB3oXIj9SpsLqZkVwJq55eOrDEvzGZgN6GRH2hq3WuYYz5zY9vPfA8ct1Remdyuqjv1Kl1p3Zakhdiq4lU9iDSxkna1oK7ASQRjCFt6O8hMXLId059KBoyun1uvRO51pKIM7lZgSL+u8sLfL6CyyHnC6iQlq2c88Fz+H8sfLXI//qWlwRBlEX2klHaY/Xg/hX4JDg+gy9
```

Comment:
This code can probably go into `webfonts.scss` now and we can finally delete `garnett.scss` 🎉!

Reply:
That is great, I will add it to the CSS refactor PR that I am preparing.